### PR TITLE
Awesome-Books: Add style 

### PR DESCRIPTION
**Pull request by mateo951**
- Added css files & css linters
- Added background image
- Added google fonts 
- Modified js file to add classes 
- Modified files to meet linters
- Added extra styling details & js classes modules

### CODE REVIEW

File: js/modules/Methods.js [hunk 1]
```diff
@@ -28,19 +28,25 @@ export default class Methods {
   }
```
Comment (carlos-ssh) [inline js/modules/Methods.js:28]: Kindly work on these changes:
- I'm facing with this error in chrome console:
Screen Shot 2021-08-11 at 19 58 09


I can't add books.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hello team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/modules/Methods.js (hunk 1 above)
Comment (alexandermorales-dev) [reply js/modules/Methods.js:28]: Hello Carlos, 

Not really sure what the issue on your side is, running this from https://mateo951.github.io/awesome-books/ should work just fine. 

Any error related to CORS it's beyond the scope of this project and our knowledge.

If you have any suggestions, they are more than welcome.

Other than that, I don't see any problem in our side.

Looking forward to your feedback.

Comment (mateo951) [reply js/modules/Methods.js:28]: Hey @alexandermorales-dev, I already talked with @carlos-ssh and he will make the review with the proper link :facepunch:

Comment (abruzy) [reply js/modules/Methods.js:28]: Hi team, I get the same CORS errors when trying to add a book as the previous said, please fix this and resubmit
Screenshot 2021-08-13 at 04 16 37
 for re-review

**Review by abruzy — CHANGES_REQUESTED** [review]
Hi Team 👋🏽, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/modules/Methods.js (hunk 1 above)
Comment (mateo951) [reply js/modules/Methods.js:28]: @abruzy You are testing it locally, please use the live demo link to test it out! https://mateo951.github.io/awesome-books/

**Comment by mateo951** [comment]
@abruzy You are testing it locally, please use the live demo link to test it out! https://mateo951.github.io/awesome-books/

**Review by krys2fa — COMMENTED** [review]
Hello @mateo951  👋,

_This is an additional review as a result of the second opinion request._

I have checked and confirmed that your application runs on the [live server](https://mateo951.github.io/awesome-books/) as you said. It is also good that you state specifically in your README that in order to run the application locally, you would need a tool like VSCode's Live Server to run it. You can go ahead to give more detail and direction in the README.

This PR should be approved by the next CR once all other requirements are met. 

Keep rocking 🚀!

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**